Trustroots/trustroots
# Improved language picking & release

#### Proposed Changes

- New language modal component, use it both in header & account page instead of having two separate ones
- Simplify 3 layers of component into one
- Add search — it shows up only when there are 8+ languages. Searches in local language as well in English. 

![image](https://user-images.githubusercontent.com/87168/110228669-83f26200-7f0b-11eb-9fd7-d4aef33a6c4a.png)


https://user-images.githubusercontent.com/87168/110228664-79d06380-7f0b-11eb-8395-beea6d8027ff.mov

In production we have fewer languages, hence also no search visible:

![image](https://user-images.githubusercontent.com/87168/110228722-0549f480-7f0c-11eb-9b47-7360b3eec5f7.png)



#### Testing Instructions

- Test in mobile
- Test searching
- You can change in `modules/core/client/utils/locales.js` "production" to "development" to see how it'll look like in production
- Test logged out header, and logged-in via `/profile/edit/account`
